cbioportal_etl/scripts/diff_studies.py

```diff
@@ -323,14 +341,24 @@ def run_py(args):
         with open(f"{comp.lower()}_current_v_update.txt", 'w') as diff_out:
             update_delta_ids, update_delta_attr_cts = catalog_and_print_delta_ids(current_data=current_data, update_data=update_data, shared_attrs=shared_attr, shared_ids=shared_ids, clin_type=comp, out=diff_out)
 
-        # Get the complete list of new and updated ids
-        updated_only_or_delta_ids: set[str] = update_only_ids | update_delta_ids
-        if len(updated_only_or_delta_ids) > 1:
-            output_delta_by_id( diff_id=updated_only_or_delta_ids, update_list=file_as_list, header=file_header, id_field=comparisons[comp]["data_table_key"], outfile_name=f"{delta_dir}/data_clinical_{comp.lower()}.txt", big_head=big_head )
+        # Create delta file outputs if update_delta_ids is populated
+        if len(update_delta_ids) > 1:
+            output_file_by_id( select_id=update_delta_ids, update_list=file_as_list, header=file_header, id_field=comparisons[comp]["data_table_key"], outfile_name=f"{delta_dir}/data_clinical_{comp.lower()}.txt", big_head=big_head )
+        # Create add file outputs if update_only_ids is populated
+        if len(update_only_ids) > 1:
+            output_file_by_id( select_id=update_only_ids, update_list=file_as_list, header=file_header, id_field=comparisons[comp]["data_table_key"], outfile_name=f"{add_dir}/data_clinical_{comp.lower()}.txt", big_head=big_head )
+            # Also process cbio_file_name_id if at sample level
+            if comp == "SAMPLE":
+                _ignore, etl_header, etl_data  = parse_file(file_path=args.manifest, header_symbol=None, column_names=True)
+                output_file_by_id( select_id=update_only_ids, update_list=etl_data, header=etl_header, id_field="Cbio_Tumor_Name", outfile_name=f"{args.study}_add_data/cbio_file_name_id.txt", big_head=_ignore )
```

Comment:
So `_ignore` is actually not going to be None here. It's going to be one line containing the column name line. If you have the `big_head` default, then you could just do this:
```suggestion
                _, etl_header, etl_data  = parse_file(file_path=args.manifest, header_symbol=None, column_names=True)
                output_file_by_id( select_id=update_only_ids, update_list=etl_data, header=etl_header, id_field="Cbio_Tumor_Name", outfile_name=f"{args.study}_add_data/cbio_file_name_id.txt" )
```